source/configure-connection.txt

```diff
@@ -0,0 +1,174 @@
+.. _django-connection-configuration:
+
+==================================
+Configure Your Database Connection
+==================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect your Django project to MongoDB
+and configure the connection.
+
+Connection Configuration 
+------------------------
+
+After installing {+django-odm+} and creating a project, you can configure
+your connection to MongoDB in the following ways:
+
+- :ref:`django-connection-configure-manual` by updating your
+  project's ``DATABASES`` setting
+- :ref:`django-connection-configure-automatic` by using
+  the ``parse_uri()`` method
+
+.. tip::
+
+   To learn how to install the Django ODM and create a
+   Django project, visit the :ref:`django-get-started` tutorial.
+
+.. _django-connection-configure-manual:
+
+Manually Configure Database Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To manually configure your connection to MongoDB, update
+the ``DATABASES`` setting in your project's ``settings.py``
+file. The ``DATABASES`` setting has a ``default`` key,
+which you must configure to set MongoDB as the default database
+connection.
+
+To configure the ``default`` key, include the following
+fields:
```

Comment:
Note to tech reviewers: not sure if "field" is the right word here? are these also keys?

Reply:
Yes `DATABASES` is a nested dictionary of settings so you might say

> To configure the ``default`` key, include the following nested `keys`.

Or

> To configure the ``default`` key, include the following settings.

Or

> To configure the ``default`` database, include the following settings.

But you probably wouldn't say "fields".